modules/cluster-logging-about-components.adoc

```diff
@@ -10,20 +10,21 @@ endif::[]
 
 :_content-type: CONCEPT
 [id="cluster-logging-about-components_{context}"]
-= About OpenShift Logging components 
+= About OpenShift Logging components
 
-The OpenShift Logging components include a collector deployed to each node in the {product-title} cluster 
-that collects all node and container logs and writes them to a log store. You can use a centralized web UI 
+The OpenShift Logging components include a collector deployed to each node in the {product-title} cluster
+that collects all node and container logs and writes them to a log store. You can use a centralized web UI
 to create rich visualizations and dashboards with the aggregated data.
 
 The major components of OpenShift Logging are:
 
-* collection - This is the component that collects logs from the cluster, formats them, and forwards them to the log store. The current implementation is Fluentd.
-* log store - This is where the logs are stored. The default implementation is Elasticsearch. You can use the default Elasticsearch log store or forward logs to external log stores. The default log store is optimized and tested for short-term storage.
+* collection - This is the component that collects logs from the cluster, formats them, and forwards them to the log store. The current implementation is the fluentd or vector collector.
+* log store - This is where the logs are stored. The default implementation was Elasticsearch, which is now deprecated. Loki is in development as the new default store. You can use the default log store or forward logs to external log stores. The default log store is optimized and tested for short-term storage.
```

Comment:
@alanconway @periklis what do we want to say here.  loki is GA.  Elasticsearch is still the "default" implementation until probably logging CR 2.0.  Does the statement about optimization and testing still hold true for Loki